.gitignore

```diff
@@ -0,0 +1 @@
+content/public-site
```

Comment:
I think we can remove this folder after copying to avoid duplicates.

Reply:
yes agreed. I was thinking it would be handy to have for testing local changes (it's gitignored) but as the build goes into the docs folder that should be sufficient. Will add that now.